tools/devworkspace-generator/src/devfile/dev-container-component-finder.ts

```diff
@@ -28,6 +28,9 @@ export class DevContainerComponentFinder {
 
     // only one, fine, else error
```

Comment:
we probably should update or remove the comment above ^

Reply:
I have fixed it